src/Helpers/CalendarUtil.php

```diff
@@ -178,21 +178,21 @@ public static function microformat($date, $time, $offset = null)
 	/**
 	 * @return array
 	 */
-	public static function get_months_map($key = '%b') 
+	public static function get_months_map($key = 'M') 
```

Comment:
are there any instances that use this method that you will also need to update

Reply:
As of the moment, there is none

Reply:
![image](https://user-images.githubusercontent.com/57464168/204694731-258ba084-4cf9-4760-94b9-bedb93c3671e.png)